op-node/cmd/genesis/cmd.go

```diff
@@ -154,31 +181,49 @@ var Subcommands = cli.Commands{
 			}
 
 			if l1Deployments != "" {
-				log.Info("L1 deployments", "path", l1Deployments)
 				deployments, err := genesis.NewL1Deployments(l1Deployments)
 				if err != nil {
-					return err
+					return fmt.Errorf("cannot read L1 deployments at %s: %w", l1Deployments, err)
```

Comment:
Was just talking to Seb about how we handle errors, I think this is a good style change.